hooks/build

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+docker build \
+	--build-arg BUILD_BASE_IMAGE=maven:3.8.2-openjdk-8 \
+	--build-arg BASE_IMAGE=openjdk:8-jre \
+	-t apache/drill:openjdk-8 \
+	-t apache/drill:latest \
```

Comment:
Just recalled the latest tag is not published for the master version. It is published for release only, so please delete it here. We use the `master` tag for builds from the master branch. The good news is that according to https://docs.docker.com/docker-hub/builds/advanced/#environment-variables-for-building-and-testing, we can use tag name from the dockerhub configuration: `DOCKER_TAG`, so please replace `latest` with that value, and add this variable to tag names with open-jdk, so we will have something like `$DOCKER_TAG-openjdk:8-jre` that will become `master-openjdk:8-jre` or `latest-openjdk:8-jre` or `1.20.0-openjdk:8-jre` for the release.